apps/ledger-live-mobile/src/screens/Swap/LiveApp/hooks/useSwapLiveAppCustomHandlers.ts

```diff
@@ -5,9 +5,9 @@ import { useMemo } from "react";
 import { useDispatch, useSelector } from "react-redux";
 import { usePTXCustomHandlers } from "~/components/WebPTXPlayer/CustomHandlers";
 import { flattenAccountsSelector } from "~/reducers/accounts";
-import { swapCustomHandlers } from "../customHandlers";
+import { earnCustomHandlers, swapCustomHandlers } from "../customHandlers";
 
-export function useSwapLiveAppCustomHandlers(manifest: LiveAppManifest) {
+export function useLiveAppCustomHandlers(manifest: LiveAppManifest) {
```

Comment:
why rename only the hook and not the file ?

And we could keep the name as is, I don't think you gain anything because it looks like it's only used in the Swap LiveApp Webview